test/e2e/framework/job/wait.go

```diff
@@ -68,7 +68,7 @@ func waitForJobPodsInPhase(ctx context.Context, c clientset.Interface, ns, jobNa
 }
 
 // WaitForJobComplete uses c to wait for completions to complete for the Job jobName in namespace ns.
```

Comment:
nit: might be useful to have the nil reason doc here as well

Reply:
That sounds reasonable. Let me add comment here.
Thank you.

Reply:
Done.